docs/connecting-via-python.adoc

```diff
@@ -0,0 +1,49 @@
+[[connecting-via-python]]
+== Connecting from Python
+
+JanusGraph can be queried from a Python application with Apache TinkerPop's http://tinkerpop.apache.org/docs/{tinkerpop_version}/reference/#gremlin-python[Gremlin-Python].
+
+Gremlin traversals can be constructed with Gremlin-Python just like in Gremlin-Java or Gremiln-Groovy. Refer to <<gremlin>> for an introduction to Gremlin and pointers to further resources.
+
+IMPORTANT: Some Gremlin step and predicate names are reserved words in Python. Those are simply postfixed with `_` in Gremlin-Python, e.g., `in()` becomes `in_()`, `not()` becomes `not_()`, and so on.
```

Comment:
I was trying to find a proper link to the reserved words, but [this](https://github.com/apache/tinkerpop/blob/3.3.4/gremlin-python/glv/generate.groovy#L29) is the closest I could find and what I used as a reference when testing the janusgraph-python driver PR. Not sure if this list is exhaustive, but if it is it would be great to include it since linking to source code is less than ideal.

Reply:
The one you linked should be the best resource as that's what is used to generate Gremlin-Python. I added the affected names to the note.